Error "use of unstable library feature" while building

**Description**
While compiling using cargo from latest master, error occurs: use of unstable library feature 'bool_to_option' in **src/config.rs:636:57**.
This error occurs in lastly changed lines: 00f52f983ce0d2162ea2fef677e2c9eee7c8f6ef

**To Reproduce**
1. sudo apt install rustc cargo libasound2-dev libssl-dev pkg-config
2. git clone https://github.com/Spotifyd/spotifyd.git
3. cd spotifyd
4. cargo build --release <- ERROR

**Expected behavior**
It should compile

**Logs**
<details><summary>Click to show logs</summary>

```
xxx@xxx:~/projects/spotifyd$ cargo build --release 
   Compiling spotifyd v0.3.3 (/home/xxx/spotifyd)
error[E0658]: use of unstable library feature 'bool_to_option'
   --> src/config.rs:636:57
    |
636 |         .map(|path| Cache::new(Some(&path), audio_cache.then_some(&path), size_limit))
    |                                                         ^^^^^^^^^
    |
    = note: see issue #80967 <https://github.com/rust-lang/rust/issues/80967> for more information

For more information about this error, try `rustc --explain E0658`.
error: could not compile `spotifyd` due to previous error

```
</details>


**Compilation flags** 
- [ ] dbus_mpris
- [ ] dbus_keyring
- [x] alsa_backend
- [ ] portaudio_backend
- [ ] pulseaudio_backend
- [ ] rodio_backend

**Versions (please complete the following information):**
<!-- DO NOT use words like `latest`. Please specify the exact version/commit hash -->
- OS: Ubuntu 20.04.5 LTS
- Spotifyd: 00f52f983ce0d2162ea2fef677e2c9eee7c8f6ef
- cargo: 1.61.0

--- a/docs/src/installation/Ubuntu.md
+++ b/docs/src/installation/Ubuntu.md
@@ -1,9 +1,16 @@
 # Ubuntu install guide
 
+## Install the rust toolchain
+
+To install the latest rust toolchain, follow the installation instructions on [rustup.rs][rustup].
+
+> **Note:** If you installed rust before via apt, you need to remove it before installing rustup.
+> We recommend to always use the latest version and don't guarantee compatibility with older ones.
+
 ## Install the requirements
 
 ```bash
-sudo apt install rustc cargo libasound2-dev libssl-dev pkg-config
+sudo apt install libasound2-dev libssl-dev pkg-config
 ```
 
 ## Clone the repository
@@ -26,3 +33,5 @@ The resulting binary will be placed in `target/release/spotifyd`
 ## Running spotifyd
 
 You can run it using `./target/release/spotifyd`
+
+[rustup]: https://rustup.rs/

